brainstormforce/astra
# AST-2362 - fix: spectra container blocks not matching frontend & backend on FW Contained on few layouts

### Description

When using spectra blocks & using FW contained, the backend & frontend experience is not the same.

### Screenshots

https://d.pr/v/dBfrHW

### Types of changes


fix: When using spectra blocks & using FW contained, the backend & frontend experience is not the same.



### How has this been tested?

- Use Bookstore Starter Template
- Edit the home page 
- select FW contained
- the backend editor experience should feel similar to frontend

### Checklist:
